protocols/exmp-sampling.md

```diff
@@ -0,0 +1,69 @@
+
+As part of this study, you will be asked to donate fecal samples of your microorganisms that will be collected on sterile cotton tipped swabs (i.e., “Q­tips”).  You will perform the sampling. Ziploc bags containing swabs will be distributed at the initial meeting (2 kits) and after that weekly from the drop off location.
+
+Samples should be taken each Sunday and Thursday following the instructions detailed below.
+
+  *	Please note: there are no back-up swabs.
+  *	If you accidentally drop the swab, or touch it to anything other than the specific sampling site, please skip that sample. The sample can be simply thrown away.
+
+You will fill out one Per Sample Event Questionnaire for every sample collected.
+
+#                 Step by step directions for fecal sampling
+
+
+You will be provided kits to collect fecal swab samples, these should be collected on Sunday and Thursday (or as close to these days as possible). Each time you collect a sample, follow these instructions. Collect each of these 2 weekly samples, one at a time, on Sunday and Thursday.
+
+  1.	From the zip­lock bag provided by the researchers, take out a labeled tube.  
+  2.	Take the cotton swab out of the labeled tube, handling the swab only by the red plastic handle.  
+  3.	Sampling protocol
+   *	You are asked to donate microorganisms from the intestinal microhabitat by collecting a fecal smear from used bathroom tissue by taking a swab of the your used tissue.
+   *	To do this, rub the swab over the used portion of the tissue until there is visible fecal material on the swab (i.e. the swab is brown).
+  4.	Return the sample swab to the labeled tube immediately after sampling and close the cap securely.
+  5.	Return the swab to the Ziploc bag.
+  6.	Be sure to wash your hands with soap and water after sampling to avoid contamination.  
+  7. Submit your sample ID via the Per Sample Event Questionnaire.
+  8. Take to the drop off point at the ARD building (room 220) at NAU within 48 hours of sampling.
+  *	If you are not able to reach a drop off location within 48 hours, store the samples in a cool (approximately room temperature) dry place out of the reach of children and indicate that it has been more than 48 hours since collection on the sample bag.
+  *	Note: these samples are unlikely to be useable and hence it will not count toward your sample count. This will affect your eligibility for compensation if this occurs more than twice.
+
+Please see Appendix A and the video link <https://vimeo.com/62393487> for additional information on how much sample to collect and proper care of the swabs. Appendix A and the video were obtained from the American Gut Project. <https://microbio.me/AmericanGut>
```

Comment:
Was the text above also derived from the American Gut Project? This attribution works as long as it's comprehensive. Can you just boldface it?